### PR TITLE
[DOC-390] Header to CSV files in SQL Ingest/Egress

### PR DESCRIPTION
**Pull request by nadegepepin**

### CODE REVIEW

File: doc/docs/2.2.x/how-tos/basic-data-operations/export-data-out-pachyderm/sql-egress.md [hunk 1]
```diff
@@ -48,6 +48,20 @@ Append an egress section to your pipeline specification file, then fill in:
 - the `url`: the connection string to your database. Its format is identical to the [url in the SQL Ingest](../../sql-ingest/#database-connection-url){target=_blank}.
 - the `file_format` type: CSV for now.
 - the `name`: the Kubernetes secret name.
+- the `columns`: Optional array for egress of **CSV files with headers only**. The order of the columns in this array must match the order of the schema columns; however, the CSV columns can be any order. So if the array is ["foo", "bar"] and the CSV file is:
```
Comment (robert-uhl) [inline doc/docs/2.2.x/how-tos/basic-data-operations/export-data-out-pachyderm/sql-egress.md:51]: I thought that JSON ingest uses the column list as well. @albscui am I wrong?

**Review by robert-uhl — COMMENTED** [review]
Two questions for Albert, but otherwise LGTM!

File: doc/docs/2.1.x/how-tos/basic-data-operations/sql-ingest.md [hunk 2]
```diff
@@ -11,28 +11,30 @@ Our **SQL ingest** tool provides a seamless connection between databases and Pac
...
+Pachyderm's SQL Ingest uses [jsonnet pipeline specs](../../pipeline-operations/jsonnet-pipeline-specs) with the following parameters to automatically create the pipelines that access, query, and materialize the results of a SQL query to a data warehouse. The outputted results can take the form of CSV or JSON files. Check the [Formatting section](#formats-and-sql-datatypes) at the bottom of the page for specific details on formats and SQL Datatypes.
 
 Pass in the following parameters and get your results committed to an output repo, ready for the following downstream pipeline:
 ```shell
 pachctl update pipeline --jsonnet https://raw.githubusercontent.com/pachyderm/pachyderm/{{ config.pach_branch }}/src/templates/sql_ingest_cron.jsonnet \
   --arg name=myingest \
   --arg url="mysql://root@mysql:3306/test_db" \
   --arg query="SELECT * FROM test_data" \
+  --arg hasHeader=false \
```
[the remark this answers is not in the document]
Comment (nadegepepin) [reply doc/docs/2.1.x/how-tos/basic-data-operations/sql-ingest.md:22]: I mention that it defaults to false. In the docs, I like to display it so people know that it exists?

File: doc/docs/2.2.x/how-tos/basic-data-operations/export-data-out-pachyderm/sql-egress.md (hunk 1 above)
Comment (nadegepepin) [reply doc/docs/2.2.x/how-tos/basic-data-operations/export-data-out-pachyderm/sql-egress.md:51]: As of today, SQL ingest uses Jsonnets. You have no obvious way to pass this column list.
The Ingest with Json uses the elements in your SELECT clause or the Aliases you are defining (i.e.: client.name as Name)

**Pull request merged**